addon/mixins/touch-action.js

```diff
@@ -45,7 +44,7 @@ export default Mixin.create({
       click
     } = this;
 
-    const hasClick = click !== K;
+    const hasClick = typeof click === 'function';
     const hasTag = (typeof tagName === 'string' && tagName.length > 0) || (tagName === null && hasClick);
```

Comment:
yeah thats what i meant haha

regarding the dirty check for the this.click - i'll try the simple way - just prop checking - if not i'll look in git for ur dirty check - it worked fine - sorry for many dumb q's :)